interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -122,8 +122,23 @@ int {{ model.name }}_acados_free_capsule(nlp_solver_capsule *capsule)
 
 
 int {{ model.name }}_acados_create(nlp_solver_capsule * capsule)
+{
+    int n_stages = {{ model.name | upper }}_N;
+    double* const_time_step = NULL; // NULL -> don't alter the exported value
+    return {{ model.name }}_acados_create_w_stages(capsule, n_stages, const_time_step);
+}
```

Comment:
I suggest to change the lines above to:
```
    int N_shooting_intervals = {{ model.name | upper }}_N;
    double* const_time_step = NULL; // NULL -> use time step from code generation
    return {{ model.name }}_acados_create_with_discretization(capsule, N_shooting_intervals, const_time_step); 
```
I think the function name: `{{ model.name }}_acados_create_with_discretization` would better describe what it does.